### PR TITLE
feat(lint): Added sass linter

### PR DESCRIPTION
**Pull request by SyntaxStacks**
Added a sass linter

### CODE REVIEW

**Comment by bighappyface** [comment]
@SyntaxStacks @jhthompson the footer has been merged into 0.5.2 and now is a good window to introduce sass lint per our talk this evening.

File: styleguide/_themes/derek/scss/components/_ceiling.scss [hunk 1]
```diff
@@ -1,143 +1,141 @@
...
       }
-      li a{
-        @extend .white;
-        &:hover {
-          background-color: lighten($signup-green, 5%);
-        }
+
+      &:focus,
+      &:active {
+        background-color: $dropdown-light-gray;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/_ceiling.scss]: dropdown-light-gray?

File: styleguide/_themes/derek/scss/components/_navigation.scss [hunk 2]
```diff
@@ -1,110 +1,129 @@
...
-  border: 0;
   background: $brand-black;
-  -webkit-border-radius: 0;
-  -moz-border-radius: 0;
+  border: 0;
   border-radius: 0;
-  border: none;
 
   .navbar-nav > li > a {
-    color: #E0E0E0;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/_navigation.scss]: someplaces we're defining new colors, some are in hex and some are lighten or darken? can we meet up to talk about the one source of truth on how colors are happening?

File: styleguide/_themes/derek/scss/components/_navigation.scss [hunk 3]
```diff
@@ -1,110 +1,129 @@
...
+    &:focus {
+      background: $inactive;
       color: $brand-black;
-      background: #ccc;
     }
   }
 
   .dropdown-menu {
-    border: #ccc 1px solid;
+    border: $inactive 1px solid;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/_navigation.scss]: inactive border?

File: styleguide/_themes/derek/scss/components/_ceiling.scss (hunk 1 above)
Comment (SyntaxStacks) [inline styleguide/_themes/derek/scss/components/_ceiling.scss]: There were many color codes w/o variable names. the added colors are just to round up all those

File: styleguide/_themes/derek/scss/components/_navigation.scss [hunk 4]
```diff
@@ -1,110 +1,129 @@
...
     background-color: $brand-primary;
+    background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/navigation/rackspace-logo-tagline-simplified.svg');
     background-position: 10px 10px;
     background-repeat: no-repeat;
     background-size: 140px 36px;
     position: relative;
     width: 160px;
+
     &:hover {
       background-color: $brand-primary;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/components/_navigation.scss]: are we going to use rackspace-red or brand-primary?

File: styleguide/_themes/derek/scss/snowflakes/_homepage.scss [hunk 5]
```diff
@@ -1,65 +1,108 @@
...
+    border-color: $mustard;
+
+    i {
+      color: $mustard;
+    }
+  }
+}
+
+.cloud-and-tech {
+  background-color: $cloud-and-tech-aqua;
```
Comment (amberRrucker) [inline styleguide/_themes/derek/scss/snowflakes/_homepage.scss]: more weird color stuff. who is even adding this?

File: styleguide/_themes/global/scss/_forms.scss [hunk 6]
```diff
@@ -24,53 +24,82 @@ p a {
...
+
+    label {
+      color: $success;
+    }
+  }
+
+  .form-control {
+    background-color: $success-background;
+    border-color: $success;
+  }
```
Comment (amberRrucker) [inline styleguide/_themes/global/scss/_forms.scss]: awesome refactor. these were leftover from original zoolander. looks so nice now.

File: styleguide/derek/_partials/solutions/key-offers.ejs [hunk 7]
```diff
@@ -7,24 +7,24 @@
       </div>
     </div>
     <div class="row row-eq-height">
-      <div class="col-md-4 col-sm-6 offerBox clickarea">
+      <div class="col-md-4 col-sm-6 offer-box clickarea">
         <%- partial("../grid/widget/key-offer", {desc: "150 max character count here. Pro gravida dolor sit amet lacus accumsan at viverra justo commodo. 150 max character count here. Pro gravida dolor sit amet lacus accumsan at viverra justo commodo. 150 max character count here. Pro gravida dolor sit amet lacus accumsan at viverra justo commodo. 150 max character count here. Pro gravida dolor sit amet lacus accumsan at viverra justo commodo."}) %>
       </div>
-      <div class="col-md-4 col-sm-6 offerBox clickarea">
```
Comment (amberRrucker) [inline styleguide/derek/_partials/solutions/key-offers.ejs]: we're going back to camel case right?

**Comment by writes** [comment]
+1

**Comment by WidgetsBurritos** [comment]
+2

**Pull request merged**